apkid/rules/dex/obfuscators.yara

```diff
@@ -241,3 +241,22 @@ rule aamo_str_enc : obfuscator
   condition:
     1 of ($opcodes*) and all of ($a, $b)
 }
+
+rule gemalto_sdk : obfuscator
+{
+  meta:
+    description = "Gemalto"
+    url         = "https://www.gemalto.com/brochures-site/download-site/Documents/eba_ezio_on_mobile.pdf"
+    author      = "Eduardo Novella"
+    sample      = "294f95298189080a25b20ef28295d60ecde27ee12361f93ad2f024fdcb5bdb0b"
+
+
+  strings:
+    $p1 = "Lcom/gemalto/idp/mobile/"
```

Comment:
Possible to build rule off of actual obfuscation? Perhaps some strings, decoder opcodes, etc?

Reply:
These three packages are good enough to detect the different Gemalto SDKs. Of course, we can add more strings, but maybe we would miss some matches.